_partials/_hypercore_policy_workflow.md

```diff
@@ -0,0 +1,115 @@
+import EarlyAccess from "versionContent/_partials/_early_access.mdx";
+
+1. **Connect to your $SERVICE_LONG**
+
+   In [$CONSOLE][services-portal] open an [SQL editor][in-console-editors]. You can also connect to your service using [psql][connect-using-psql].
+
+1. **Enable columnstore on a hypertable**
+
+   Create a [job][job] that automatically moves chunks in a hypertable to the columnstore at a specific time interval. 
+   By default, your table is `orderedby` the time column. For efficient queries on columnstore data, remember to
+   `segmentby` the column you will use most often to filter your data:
+
+   * [Use `ALTER TABLE` for a hypertable][alter_table_hypercore]
+     ```sql
+     ALTER TABLE stocks_real_time SET (
+        timescaledb.enable_columnstore = true, 
+        timescaledb.segmentby = 'symbol');
+     ```
+   * [Use ALTER MATERIALIZED VIEW for a continuous aggregate][compression_continuous-aggregate]
+     ```sql
+     ALTER MATERIALIZED VIEW stock_candlestick_daily set (
+        timescaledb.enable_columnstore = true, 
+        timescaledb.segmentby = 'symbol' );
+     ``` 
+     Before you say `huh`, a continuous aggregate is a specialized hypertable.
+   
+   * <EarlyAccess /> Enable indexing over all data in the rowstore and columnstore:
+     
+     ```sql
+     alter table stocks_real_time,
+        set access method hypercore,
+        set (timescaledb.enable_columnstore = true, timescaledb.segmentby = 'symbol');
+     ```
+     This is also early access for continuous aggregates.
+   
+1. **Add a policy to move chunks to the columnstore at a specific time interval**
+
+   For example, 60 days after the data was added to the table:
+   ``` sql
+   CALL add_columnstore_policy('older_stock_prices', after => INTERVAL '60d');
+   ```
+   See [add_columnstore_policy][add_columnstore_policy].
+
+   * <EarlyAccess /> To enable indexing over data in the rowstore and the columnstore, tell the policy 
+     to use the Hypercore table access method.
+   
+      ``` sql
+      CALL add_columnstore_policy(
+         'older_stock_prices', 
+         after => INTERVAL '60d',  
+         hypercore_use_access_method => true);
+      ```
```

Comment:
Again, I don't think this belongs in here. This should be listed with other indexing options and explaining tradeoffs / when to use not to use. We should NOT guide everyone to enable this.

Reply:
Done